src/test/data/JsonSerializableAssignmentMapTest/invalidAssignmentMap.json

```diff
@@ -0,0 +1,84 @@
+{
+  "persons": [ {
+    "name": "Valid Person Invalid AssignmentMap",
+    "phone": "9482424",
```

Comment:
If we include bonus xp, it will be like 900/800
Will this be invalid?

Reply:
Bonus XP is up to +75, will include it in the next commit